### PR TITLE
Entity sequence

### PR DESCRIPTION
**Pull request by anthonysena**
Continuation of work from @wivern as part of #324.

Add fix for Flyway migration for PostgreSQL. With this fix applied, the changes look good from my side for PostgreSQL. I've also added a commit to address a regression in another SQL Server flyway script for dropping the password/salt fields for the sec_user table per @chrisknoll's issue raised on: https://github.com/OHDSI/WebAPI/pull/327#pullrequestreview-97933012

Tagging @chen-regen for awareness and help testing on Oracle.

### CODE REVIEW

**Comment by anthonysena** [comment]
Noting that this is being tested by @pavgra and team on Oracle with sample data to ensure the entity sequencing does not trash any existing data.

File: src/main/resources/db/migration/postgresql/V2.2.5.20180215105415__separate-sequences.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+SELECT setval('${ohdsiSchema}.cohort_definition_sequence', coalesce(max(id), 1)) FROM ${ohdsiSchema}.cohort_definition;
+
+-- concept_set_sequence already exists - no need to create
+SELECT setval('${ohdsiSchema}.concept_set_sequence', coalesce(max(concept_set_id), 1)) FROM ${ohdsiSchema}.concept_set;
+
+-- concept_set_item_sequence already exists - no need to create
+SELECT setval('${ohdsiSchema}.concept_set_item_sequence', coalesce(max(concept_set_item_id), 1)) FROM ${ohdsiSchema}.concept_set_item;
+
+-- concept_set_negative_controls_sequence already exists - no need to create
+SELECT setval('${ohdsiSchema}.concept_set_negative_controls_sequence', coalesce(max(id), 1)) FROM ${ohdsiSchema}.concept_set_negative_controls;
```
Comment (mpozhidaeva) [inline src/main/resources/db/migration/postgresql/V2.2.5.20180215105415__separate-sequences.sql]: @wivern - I guess 'negative_controls_sequence'- not 'concept_set_negative_controls_sequence'?

**Comment by mpozhidaeva** [comment]
@anthonysena, @pavgra, @wivern - I added some fixes after testing the oracle migration, it would be great if you review them. Also pay attention to the attached file with the states of the modified tables (before and after the migration).
[oracle-migration-test.zip](https://github.com/OHDSI/WebAPI/files/1866643/oracle-migration-test.zip)

**Comment by mpozhidaeva** [comment]
@anthonysena, if you agree with changes and there are no questions - let's merge the pull request:)

**Comment by anthonysena** [comment]
@mpozhidaeva these changes look good from my side - thanks for including the before/after states for the tables impacted by this change. Since I opened this PR (based on the work your team did in the other fork) can you approve? I'll then get it merged into master. Thanks!

**Pull request merged**